index.js

```diff
@@ -49,7 +49,11 @@ function makeId(length) {
         req.originalUrl = u;
         next();
       }
-    } else {
+    }else if(/^\/login/.test(o.pathname)){
```

Comment:
Why handle this route by redirecting? Since it is not doing anything, can we get rid of it?

Reply:
This will be removed after the new discord update. In which login has been removed.